packages/@webex/plugin-meetings/src/locus-info/parser.ts

```diff
@@ -716,10 +723,16 @@ export default class Parser {
       // wait for desync response
       this.pause();
     } else if (lociComparison === USE_INCOMING) {
+      const locusUrlChanged = this.workingCopy?.url && newLoci?.url !== this.workingCopy.url;
+
       // update working copy for future comparisons.
       // Note: The working copy of parser gets updated in .onFullLocus()
       // and here when USE_INCOMING locus.
       this.workingCopy = newLoci;
+
+      if (locusUrlChanged) {
+        this.triggerSync('locus url changed, likely due to breakout session move');
```

Comment:
looking at this a bit more I'm not sure if this is correct... we call `triggerSync()` here, but we will still call `this.onDeltaAction()` in line 768, so we will start processing the incoming delta and applying it to our Locus DTO copy **before** the sync is completed, so it will mess things up. 

This makes me think that maybe we should also take a different approach and not use USE_INCOMING but have a new value added to `LocusDeltaParser.loci`, something like `LocusDeltaParser.loci.LOCUS_URL_CHANGED` and return that in `compareDelta()` so that it's more clear what's happening and then in `processDeltaEvent()` we would handle the new value and just update the `this.workingCopy` and in `applyLocusDeltaData()` for the new value just do the same we do for `DESYNC` there, because by that point the workingCopy.url would be already updated to the right one. What do you think?

Reply:
I like the suggestion.  Just pushed the rework.